fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -38,12 +38,13 @@ std::unordered_map<SpecialKeySpace::MODULE, KeyRange> SpecialKeySpace::moduleToB
 // orEqual == false && offset == 1 (Standard form)
 // If the corresponding key is not in the underlying key range, it will move over the range
 ACTOR Future<Void> moveKeySelectorOverRangeActor(const SpecialKeyRangeBaseImpl* skrImpl, ReadYourWritesTransaction* ryw,
-                                                 KeySelector* ks) {
+                                                 KeySelector* ks, Optional<Standalone<RangeResultRef>>* cache) {
```

Comment:
Maybe add a short comment in the actor to explain how param cache is used?

Reply:
Yeah, right. I should do it.